helpers/helper-translation-discussion.php

```diff
@@ -735,6 +735,24 @@ public static function get_reject_reasons() {
 		);
```

Comment:
Instead of having the new `get_reject_reason_explanations` function, I'm looking at the possibility of modifying the  array returned by `get_reject_reasons` to having the array key as the reason text and the explanation as array value. 
```
return array(
	'style_guide' => __( 'The translation is not following the style guide. It will be interesting to provide a link to the style guide for your locale in the comment.' ),
	'grammar'     => __( 'The translation has some grammar problems. It will be interesting to provide a link explaining the grammar issue for your locale in the comment.' ),
```
When we want to display the reason text, we can always replace the underscore `_` with a space and capitalize first letter of each word.

Reply:
If we develop this approach, we can't translate the rejection reason, as is the key in the array. I prefer the current approach, because it is cleaner. We have 2 different arrays, using the same key, and we get the info always from the array value.

Reply:
Done!